main.go

```diff
@@ -1,109 +1,19 @@
 package main
 
 import (
-	"database/sql"
-	"errors"
-	"fmt"
 	"log"
-	"strings"
 
-	"github.com/gobuffalo/packr"
-
-	"github.com/spf13/cobra"
-
-	_ "github.com/go-sql-driver/mysql"
+	"github.com/LUSHDigital/modelgen/cmd"
 )
 
 var (
-	output   *string
-	dbName   *string
-	pkgName  *string
-	conn     *string
-	database *sql.DB
-	version  string
-	box      packr.Box
+	version string
 )
 
 func init() {
```

Comment:
Could we get rid of this `init`?

I'd much rather we define a setup package/flow instead, so we can control it better and be consistent.